services/supervisor/src/transactions.rs

```diff
@@ -183,12 +193,14 @@ impl SupervisorInterface for Supervisor {
 
         let propose_hash = propose.object_hash();
         config_confirms.confirm(&propose_hash, author);
-
-        let config_entry = ConfigProposalWithHash {
-            config_propose: propose,
+        let actual_from = propose.actual_from;
+        let config_info = ConfigProposalInfo {
+            author,
+            config_propose: propose.clone(),
```

Comment:
Also you can rename variable to the `config_propose` and have a clean constructor call as a result.